Arrays/Reversing-Encryption.java

```diff
@@ -0,0 +1,29 @@
+import java.util.*;
+
+public class Main {
+    private static Scanner sc = new Scanner(System.in);
+
+    public static void main(String args[]) {
```

Comment:
Please state the Questions URL or full question on top of the file

Reply:
Please follow camelcase in file name

Reply:
Pasted the problem url on the top of the file
Name changed to CamelCase